### PR TITLE
Basic TravisCI Config

### PR DESCRIPTION
**Pull request by SpainTrain**
Super simple .travis.yml for running tests on node 4 and 0.12.  You will likely need to add the project at https://travis-ci.org/. Inspired by really wanting my PR tested on CI (https://github.com/faassen/reselect/pull/49) X-D

You will need this file anyway for https://github.com/faassen/reselect/issues/15 ;-)

[Review on Reviewable](https://reviewable.io/reviews/faassen/reselect/50)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+language: node_js
+node_js:
+      - "0.12"
+      - "4.0.0"
+script:
+      - npm run lint
+      - npm test
+      - npm run test:cov
```
Comment (SpainTrain) [inline .travis.yml]: just to see coverage report in CI output

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+      - npm run compile
```
Comment (SpainTrain) [inline .travis.yml]: basic sanity check, maybe superfluous

**Comment by ellbee** [comment]
Thanks @SpainTrain, looks good!

@faassen I can't add Reselect to travis, can you do this when you get a moment please?

**Comment by faassen** [comment]
Okay, reselect should be part of travis now.

**Comment by ellbee** [comment]
Yay! Cheers @SpainTrain and @faassen!

**Comment by SpainTrain** [comment]
:tada:

**Pull request merged**